directjngine/src/main/java/com/softwarementors/extjs/djn/EncodingUtils.java

```diff
@@ -25,10 +25,20 @@
 
 package com.softwarementors.extjs.djn;
 
+import org.apache.commons.lang.StringEscapeUtils;
+
 public final class EncodingUtils {
   private EncodingUtils() {
     // Avoid instantiatin
   }
   
   public static final String UTF8 = "UTF-8";
+
+  public static String htmlEncode(String value) {
+    if (value == null) {
+      return value;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') com.softwarementors.extjs.djn.EncodingUtils.htmlEncode(String) may return null, but is declared @Nonnull [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.s/coding_rules#rule_key=findbugs%3ANP_NONNULL_RETURN_VIOLATION)